pkg/gofr/datasource/file/ftp/mock_logger.go

```diff
@@ -0,0 +1,49 @@
+package ftp
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+// Level represents different logging levels.
+type Level int
+
+const (
+	DEBUG Level = iota + 1
+	INFO
+	ERROR
+)
+
+type MockLogger struct {
```

Comment:
This is now an exported type that is not limited to testing… not sure we want to pollute the public API with a mocked implementation?

But not only that, this isn’t a mock logger, because this is actually printing out log messages?